tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -406,6 +405,9 @@ public void testWebsocketClientInWebappProvidedByServer(String scheme) throws Ex
             String[] args2 = {
                 "jetty.http.port=" + port,
                 "jetty.ssl.port=" + port,
+                // We need to expose the websocket client classes to the webapp for this to work.
+                "jetty.webapp.addServerClasses+=,-org.eclipse.jetty.websocket.client.",
+                "jetty.webapp.addSystemClasses+=,+org.eclipse.jetty.websocket.client.",
```

Comment:
If the server is to provide this, then any exposing should be done in a Configuration... off by default, but that is added to a webapp if we want it to see the client.

Reply:
Do I need to write a class? 

I thought a Jetty module should be enough, and the test mimics what should be in the `*.mod` file if I want the Jetty WebSocket client classes be provided by the server.